geomstats/geometry/symmetric_matrices.py

```diff
@@ -5,11 +5,11 @@
 import geomstats.backend as gs
 import geomstats.vectorization
 from geomstats import algebra_utils
-from geomstats.geometry.embedded_manifold import EmbeddedManifold
+from geomstats.geometry.embedded_manifold import EmbeddedManifold, OpenSet
 from geomstats.geometry.matrices import Matrices
 
 
-class SymmetricMatrices(EmbeddedManifold):
+class SymmetricMatrices(EmbeddedManifold, OpenSet):
```

Comment:
Why do we need OpenSet here? It seems that its methods are not called?

Reply:
Its a particular case because this is a vector space. I thought OpenSet would be useful for tangent vecs?

Reply:
Gooot it, for `is_tangent`, `to_tangent`, etc. Thanks! (see my other comment on `is_tangent`, `to_tangent` and the fact that we have to impose that the ambient manifold is a vector space with the current implementation)